client/src/services/recentFiles.ts

```diff
@@ -0,0 +1,52 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { DetectedFileType } from '@/common/fileTypes';
+import { EntryID, EntryName, FsPath, WorkspaceHandle } from '@/parsec';
+import { Ref, ref } from 'vue';
+
+interface RecentFile {
+  entryId: EntryID;
+  workspaceHandle: WorkspaceHandle;
+  path: FsPath;
+  name: EntryName;
+  contentType?: DetectedFileType;
+}
+
+const HISTORY_SIZE = 10;
```

Comment:
Should probably be a config variable (user preferences) and 5 by default seems more fair.

Reply:
Same, env variables are defined at packaging time, and we can add it as a user preferences later if needed.